views/layouts/main.hbs

```diff
@@ -24,18 +24,19 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="/css/main.css">
 </head>
-<body class="relative">
+<body>
 
     <a href="https://github.com/jenkoian/hacktoberfest-checker" target="_blank" class="absolute top-0 right-0"><img src="https://camo.githubusercontent.com/652c5b9acfaddf3a9c326fa6bde407b87f7be0f4/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6f72616e67655f6666373630302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png"></a>
 
-    <div class="mw9 center ph3-ns">
+    <div class="mw9 center ph3-ns main-content">
```

Comment:
Add `ma0` for the margin. Don't think tachyons supports flex-grow/shrink so will likely need this in css.